.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!---
+Thank you for your contribution. Please make sure your pull request fulfils all of the below requirements. If you cannot currently tick all the boxes, but would still like to create a PR, then add the label "work in progress" and assign the PR to yourself.
+--->
+
+- [ ] Each commit message has a non-empty body, answering the "Why?" question.
+- [ ] My contribution is formatting in line with CODING_STANDARD.md.
+- [ ] Methods or procedures I have added are documented.
```

Comment:
...using Doxygen comments at the definition site (not the declaration)

Reply:
I have included a reference to CODING_STANDARD.md.